test/internal/sanctuary.js

```diff
@@ -17,8 +17,8 @@ function UnaryType(typeIdent) {
 
 //  env :: Array Type
 var env = S.env.concat([
-  UnaryType('sanctuary/Compose'),
-  UnaryType('sanctuary/Identity')
+  UnaryType('sanctuary/Compose')($.Unknown),
+  UnaryType('sanctuary/Identity')($.Unknown)
```

Comment:
It looks like the purpose of `UnaryType` is to mock a type for the environment. So I think you may as well apply `$.Unknown` to the type inside the `UnaryType` function, changing its signature to `String -> Type`.